source/tests/pt/model/test_dipole_fitting.py

```diff
@@ -0,0 +1,277 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import itertools
+import unittest
+
+import numpy as np
+import torch
+from scipy.stats import (
+    special_ortho_group,
+)
+
+from deepmd.dpmodel.fitting import DipoleFitting as DPDipoleFitting
+from deepmd.pt.model.descriptor.se_a import (
+    DescrptSeA,
+)
+from deepmd.pt.model.task.dipole import (
+    DipoleFittingNet,
+)
+from deepmd.pt.utils import (
+    env,
+)
+from deepmd.pt.utils.nlist import (
+    extend_input_and_build_neighbor_list,
+)
+from deepmd.pt.utils.utils import (
+    to_numpy_array,
+)
+
+from .test_env_mat import (
+    TestCaseSingleFrameWithNlist,
+)
+
+dtype = env.GLOBAL_PT_FLOAT_PRECISION
+
+
+class TestDipoleFitting(unittest.TestCase, TestCaseSingleFrameWithNlist):
+    def setUp(self):
+        TestCaseSingleFrameWithNlist.setUp(self)
+
+    def test_consistency(
+        self,
+    ):
+        rng = np.random.default_rng()
+        nf, nloc, nnei = self.nlist.shape
+        dd0 = DescrptSeA(self.rcut, self.rcut_smth, self.sel).to(env.DEVICE)
+        rd0, gr, _, _, _ = dd0(
+            torch.tensor(self.coord_ext, dtype=dtype, device=env.DEVICE),
+            torch.tensor(self.atype_ext, dtype=int, device=env.DEVICE),
+            torch.tensor(self.nlist, dtype=int, device=env.DEVICE),
+        )
+        atype = torch.tensor(self.atype_ext[:, :nloc], dtype=int, device=env.DEVICE)
+
+        for distinguish_types, nfp, nap in itertools.product(
+            [True, False],
+            [0, 3],
+            [0, 4],
+        ):
+            ft0 = DipoleFittingNet(
+                "foo",
+                self.nt,
+                dd0.dim_out,
+                3,
+                dim_rot_mat=100,
```

Comment:
```suggestion
                dim_rot_mat=dd0.get_dim_emb(),
```
The same suggestion for other `dim_rot_mat`